fnmatch/fnmatch.py

```diff
@@ -26,10 +35,10 @@ def fnmatch(name, pat):
     [seq]   matches any character in seq
     [!seq]  matches any char not in seq
 
-    An initial period in FILENAME is not special.
+    An initial period in FILENAME is not special
```

Comment:
Unrelated formatting would better go in separate sommits. Thoug I'm not sure this is even need?

Reply:
I didn't specifically add the 'period' at the end; but it could have been a typo on my part.